src/weiss_gripper_ieg76/driver.py

```diff
@@ -17,14 +17,27 @@ def __init__(self):
 		self.states_publisher_thread = StatesPublisher(0.8, self.serial_port_comm)
 		rospy.on_shutdown(self.shutdown_handler)
 
+	def set_max_pos(self, gripper_info):
```

Comment:
So do we support all those grippers? We should add this to the description and change the name in a future PR?

Reply:
added some reference in readme